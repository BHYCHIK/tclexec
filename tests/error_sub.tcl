set b 10
set a [
  set b
  set b; set a
]

