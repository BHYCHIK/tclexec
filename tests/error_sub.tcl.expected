Line: 4, position: 14: variable a doesn't exist
  set b; set a
             ^
